### DISCUSSION TITLE
How to have custom icon.

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [zed-industries/zed#8711] [Feature Requests]
On my VSCode 💩 I use [`material-icon-theme`](https://marketplace.visualstudio.com/items?itemName=PKief.material-icon-theme) to have cool icon and more visible. 
Is it possible to do the same with zed? Because that's one of the things that really bothers me.

**Comment by Moshyfawn** [comment]
Zed doesn't currently support custom file icons. #4609 is one of the more prevalent icon customisation issues. The Zed team is big on clean, minimalist designs, and unless the extensions support this in the future _(which I think I've heard they will)_, I doubt Zed will add this functionality to their core codebase.

**Reply by AugustinMauroy** [reply]
Thanks for answer
